woocommerce/woocommerce-gateway-amazon-pay
# Adds support for Amazon Pay on Cart and Checkout Blocks of WooCommerce Blocks

### All Submissions:






### Changes proposed in this Pull Request:

Adds compatibility support for WooCommerce Blocks. 



Closes #171 .

### How to test the changes in this Pull Request:

1. All button placements should be compatible with a checkout and cart created using WooCommerce Blocks.
Placements include: 

- on mini-cart
- on single-product
- on cart
- on checkout
- on the order-pay endpoint

### Other information:




### Changelog entry

> Add - Amazon Pay gateway support for WooCommerce Blocks.
